app/templates/components/schema-field-radio.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="form-group">
+  {{#if property.title}}
+    <label for="{{key}}">{{property.title}}</label>
+  {{/if}}
+  <div class="radio">
+    <div>
+      {{#radio-button value=false groupValue=value name=key changed="changed"}}
```

Comment:
Is `key` here the full document path, or just the last section?  If it is just the last portion, then these may not be uniquely named if there are many onscreen at once...

Also, what is `groupValue`? It seems from the component's `init` that this will be `true` or `false`, and unless I misunderstand how this `radio-button` component works this means that all true or false radio buttons will share the same group (which will mean that they are mutually exclusive across properties and not just between true and false of the same property).